http/handler.go

```diff
@@ -34,7 +34,7 @@ func Handler(core *vault.Core) http.Handler {
 	mux.Handle("/v1/sys/seal", handleSysSeal(core))
 	mux.Handle("/v1/sys/step-down", handleSysStepDown(core))
 	mux.Handle("/v1/sys/unseal", handleSysUnseal(core))
-	mux.Handle("/v1/sys/renew/", handleLogical(core, false, nil))
+	mux.Handle("/v1/sys/renew", handleLogical(core, false, nil))
```

Comment:
Will this not route the `/v1/sys/renew/<urlparam>` to `handleLogical` with `dataOnly` set to `true`?